q2_mlab/doctor.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+import click
+import os
+from datetime import datetime
+
+
+def parse_info(info_filepath):
+    with open(info_filepath) as f:
+        lines = f.readlines()
+        info_dict = {
+            key.strip(): val.strip()
+            for key, val in zip(lines[0].split(","), lines[1].split(","))
+        }
+    return info_dict
+
+
+def get_uninserted_results(results_dir):
```

Comment:
maybe rename this function? Iirc it might be possible for an artifact to exist at this path but also have been inserted already, e.g., if a job fails between inserting the artifact and moving the qza

Reply:
Good point. Since I'm going to use this function elsewhere I generalized it to just get_results and added a docstring.